Sources/Filters/General/TubeFilter/index.js

```diff
@@ -707,6 +707,21 @@ function vtkTubeFilter(publicAPI, model) {
       output.getPointData().addArray(newArray); // concat newArray to end
     }
 
+    // loop over cellData arrays and resize based on numNewCells
+    const numNewCells = inLines.getNumberOfCells() * model.numberOfSides + 2;
```

Comment:
Why +2 ?

Reply:
The +2 is to account for capping. Though I can probably save that extra 2 cell slots if I check to see if capping is enabled.